src/Sticky/index.jsx

```diff
@@ -41,10 +41,11 @@ const Sticky = React.forwardRef(({
       observer.observe(resolvedRef.current);
 
       return () => {
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Eslint suggests that we should store `resolvedRef.current` as a variable somewhere inside the hook because ref's value might change before this function will actually be called, that's why it is better to remember the value. Maybe we could do it instead of disabling the rule here?